frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -485,6 +516,26 @@ export class Libre311ServiceImpl implements Libre311Service {
 		return ServiceDefinitionSchema.parse(res.data);
 	}
 
+	async createService(params: CreateServiceRequest): Promise<CreateServiceResponse> {
+		let data;
+
+		try {
+			const res = await this.axiosInstance.post<unknown>(
+				ROUTES.postService(this.jurisdictionConfig),
+				params
+			);
+
+			if (res.data) {
+				data = res.data[0 as keyof typeof unknown];
```

Comment:
I don't think this if block is needed.  The parse method should properly narrow the type.